includes/Data/ContentBlocksResolver.php

```diff
@@ -66,9 +66,8 @@ function ( $parsed_block ) {
 			},
 			$parsed_blocks
 		);
-
 		// Flatten block list here if requested or if 'flat' value is not selected (default)
-		if ( !isset( $args['flat'] ) || 'true' == $args['flat'] ) {
+		if ( !isset( $args['flat'] ) || 'false' == $args['flat'] ) {
```

Comment:
Is this correct? I know the default behavior got changed in a prev PR, but I think `$args['flat']` still refers to a user _wanting_ to flatten the array.